src/components/TagEditDialog.vue

```diff
@@ -16,7 +16,7 @@
                         </div>
 
                         <div class="mb-3">
-                            <label for="tag-color" class="form-label">{{ $t("Color") }}</label>
+                            <label for="tag-color" class="form-label">{{ $t("color") }}</label>
```

Comment:
It should be uppercase.

Reply:
I modified it to correspond to the "color" already present in the translation files, knowing that other files use it and that just below another line does not use the capital letter
![chrome_C5DkSJX67s](https://user-images.githubusercontent.com/70776486/213741999-4a277060-c07c-4b97-9433-3c5164b46d6d.png)

Reply:
should I add the capital letter in the en.js file so that the capital letter is present?
![Code_Sh2qT68bXy](https://user-images.githubusercontent.com/70776486/213743651-bce1ebc1-47f0-47cf-bc0e-bb28e52c3086.png)

Reply:
@louislam ?

Reply:
I capitalize it in the en.js instead.